lighthouse-core/test/scripts/run-mocha-tests.js

```diff
@@ -288,6 +292,17 @@ function runMochaCLI(tests) {
     ...baseArgs,
     ...tests,
   ];
+
+  // Limit on windows is 8192 characters. This includes the invocation of the shell program
+  // (typically `C:\Windows\system32\cmd.exe /d /s /c "npx mocha ..."`), which we can't account
+  // for simply. Instead, use a limit of 8000.
+  if (process.platform === 'win32' && [file, ...args].join(' ').length > 8000) {
+    const halfPoint = Math.floor(tests.length / 2);
+    runMochaCLI(tests.slice(0, halfPoint));
+    runMochaCLI(tests.slice(halfPoint));
+    return;
+  }
+
   console.log(
     `Running command: ${argv.update ? 'SNAPSHOT_UPDATE=1 ' : ''}${file} ${args.join(' ')}`);
```

Comment:
if we invoke mocha through its JS api instead of a child-process to use its CLI.... would it fix this ? (node --inspect support would also come back )

Reply:
yup, but it's something that must be done after all the isolation stuff is ripped out. on the todo list